src/reflection/skrypnyk/PersonFactory.java

```diff
@@ -23,15 +23,17 @@ public static Person createPerson(String name, int age, String address) throws I
             throw new RuntimeException(e);
         }
     }
-    public static void printFields(Person person){
+    public static void printFields(Person person) throws IllegalAccessException {
         Class<? extends Person> personInfo = person.getClass();
         Field [] fieldName = personInfo.getDeclaredFields();
         for (Field declaredField : fieldName) {
             declaredField.setAccessible(true);
+
+            declaredField.getName();
+            Object o = declaredField.get(person);
+            System.out.println(o);
```

Comment:
Good, but, if you print it, you will get only value of field.
You need to print name as well.

sout(declaredField.getName() + ": " + o)

"o" -- is your object (Object o = declaredField.get(person);)